syntax = "proto3";

package proto;

enum ProcessType {
    POS_EXTRAPOLATOR = 0;
    LIDAR_PROCESSING = 1;
    CAMERA_PROCESSING = 2;
}

message StartupMessage {
    repeated ProcessType py_tasks = 1;
    string json_config = 2;
}

message MessageRetrievalConfirmation {
    bool received = 1;
}
